src/oscar/static_src/oscar/css/dashboard.css

```diff
@@ -0,0 +1,11286 @@
+/*!
```

Comment:
This file should not be committed - it should be compiled into `static/` at build time. Same for everything in `static_src/oscar/fonts`.